src/main/java/eriks/csa/api/OPAAdminResource.java

```diff
@@ -26,10 +27,18 @@ public class OPAAdminResource {
     @Inject
     CSAService csaService;
 
+    @Path("/member/save-angel")
+    @PUT
+    public Response saveAngel(AngelDtoIn angelDtoIn, @Context HttpHeaders headers) {
+        //validateAdminKey(headers.getHeaderString("adminKey")); #todo add this later
```

Comment:
I'll uncomment before merging  @eriksimoesoliveira could you give me the key to add on the roletão extension ?